image.go

```diff
@@ -20,6 +20,12 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Config returns the ocispec descriptor for the image
+	Config(ctx context.Context) (ocispec.Descriptor, error)
```

Comment:
We already have a type that does all this. Let's not make the interfaces so huge that the become unwieldy.